src/mod/save/data/box/box.h

```diff
@@ -5,6 +5,20 @@
 #include "externals/PlayerWork.h"
 #include "externals/System/Primitives.h"
 #include "externals/System/String.h"
+#include "externals/Pml/PokePara/SerializedPokemonFull.h"
+
+static const int64_t VANILLA_BOXSIZE = 40;
+static const uint64_t INIT_WALLPAPER_OFFSET = 8;
+static const nn::vector<const char*> boxDefaultStrings {
```

Comment:
We should read this from the actual label files, I'm worried about the memory usage here

Reply:
We'll do that later, seems like there's something we're missing and it doesn't matter enough if the vector is only in one method's scope